cypress/integration/regression/decision-tree-regression.spec.ts

```diff
@@ -0,0 +1,488 @@
+/* eslint-disable no-undef */
+import { datasetPage } from '../../page-objects/app/datasets-page'
+import { decisionTreesPage } from '../../page-objects/app/decision-trees-page'
+
+describe('Regression test of the decision tree', () => {
```

Comment:
What's the difference between this and the previous suites? I don't understand.

Reply:
There is a limit of 500 lines of code in one file, so I had to split one test into 2 spec files